File: scaffold/src/main/java/top/xuqingquan/web/publics/JsAccessEntrace.java
package top.xuqingquan.web.publics;

public interface JsAccessEntrace extends QuickCallJs {

    void callJs(String js, android.webkit.ValueCallback<String> callback);

    void callJs(String js, com.tencent.smtt.sdk.ValueCallback<String> callback);

    void callJs(String js);

}

File: scaffold/src/main/java/top/xuqingquan/web/publics/JsAccessEntrace.kt
package top.xuqingquan.web.publics

import android.webkit.ValueCallback
import com.tencent.smtt.sdk.ValueCallback as X5ValueCallback

interface JsAccessEntrace : QuickCallJs {

    fun callJs(js: String?, callback: ValueCallback<String>?)

    fun callJs(js: String?, callback: X5ValueCallback<String>?)

    fun callJs(js: String?)

}
